src/components/listings/MapLocationListing.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import {
+  ImageBackground,
+  View,
+  Text,
+  Pressable,
+  GestureResponderEvent,
+} from 'react-native'
+import { MapLocationListingItem } from '@/types/location'
+
+interface MapLocationListingProps {
+  locations: MapLocationListingItem[]
+  onPress: (event: GestureResponderEvent) => void
```

Comment:
should this be passed the singular `MapLocationListingItem` instead of the event?

Reply:
👍